misc/determine-version.py

```diff
@@ -1,11 +1,13 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 
 from __future__ import print_function
 import re
 import subprocess
 import sys
 import os
 from os.path import basename
+import  functools
+from functools import cmp_to_key
```

Comment:
Don't import functools twice. Omit line 9.

Reply:
You are correct.  Still a python neophyte.